### PR TITLE
Added info time-delayed proxies

### LINKED ISSUE #4048: Add more information on time-delay proxies
Reading through the section on time-delay proxies ( https://wiki.polkadot.network/docs/learn-proxies#time-delayed-proxy ) , it looks like you can just issue a call via the time-delay and it will occur after the time delay.

We should be explicit that there is a separate process for interacting with accounts via time-delay proxy than regular proxies, and it requires two steps (announcing and making the call).

Namely,

1. Create the extrinsic, calling from the target (not the proxy)
2. Get the hash of the extrinsic call
3. Call proxy.announce(HASH) from the proxy
4. Wait for the right number of blocks (as set when creating the proxy)
5. Call proxy.proxyAnnounced with the actual extrinsic set (like a regular proxy call, but set the delegate to the proxy and "real" to the target)

### PR DESCRIPTION
**Pull request by filippoweb3**
- added new video tutorial
- more details in the text

### CODE REVIEW

File: docs/learn/learn-proxies.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ slug: ../learn-proxies
 
 import RPC from "./../../components/RPC-Connection"
 
+:::caution The Account Tab in the Polkadot-JS UI cannot handle complicated proxy setups
+
+The Account Tab in the Polkadot-JS UI cannot handle complicated proxy setups (e.g. a proxy ->
```
Comment (profradha) [inline docs/learn/learn-proxies.md]:
```suggestion
The Accounts Tab in the Polkadot-JS UI cannot handle complex proxy setups (e.g. a proxy ->
```

File: docs/learn/learn-proxies.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ slug: ../learn-proxies
...
+multisig -> an anonymous proxy which is part of another multisig). These complex setups must be done
+using the [Extrinsics Tab](https://polkadot.js.org/apps/#/extrinsics) directly.
+
+These complex proxy setups should only be performed if you are comfortable enough interacting
+directly with the chain. **We recommend to use the Westend Testnet if you are testing features for
+the first time.**
```
Comment (profradha) [inline docs/learn/learn-proxies.md]:
```suggestion
 **We recommend to use the [Westend Testnet](learn-DOT.md#getting-tokens-on-the-westend-testnet) if you are testing features for the first time.** By performing the complex proxy setups on the testnet, you can comfortably replicate the procedure on the main networks.
```

File: docs/learn/learn-proxies.md [hunk 3]
```diff
@@ -205,23 +217,31 @@ and the `ProxyDepositFactor` is
 
 ## Time-delayed Proxy
 
+:::info
+
+See [this video tutorial](https://youtu.be/3L7Vu2SX0PE) to learn how you can use time-delayed
```
Comment (profradha) [inline docs/learn/learn-proxies.md]:
```suggestion
See [this video tutorial](https://youtu.be/3L7Vu2SX0PE) to learn how you can setup and use time-delayed
```

**Review by DrW3RK — APPROVED** [review]
Looks good. Minor suggestions included.

**Comment by filippoweb3** [comment]
@DrW3RK I did a new video, and added a figure and description. Let me know what you think!

File: docs/learn/learn-proxies.md [hunk 4]
```diff
@@ -208,20 +220,38 @@ and the `ProxyDepositFactor` is
...
+
+:::
+
+Let's take for example the stash account Eleanor that has a controller Charly. Eleanor does not
+fully trust Charly, and as a consequence sets him as a time-delayed staking proxy. In this way, if
+Charly submits an extrinsic to change the controller to Bob, such extrinsic can be rejected by
+Eleanor. Bob can be even more malicious than Charly and change the reward destination to another
+account he only controls. Remember that Eleanor added Charly as proxy, but she might not want to add
+Bob as a controller. This implies that Eleanor monitors Charly, and that within the time-delay she
+can spot the announced extrinsic. As a monitor tool you can use a block explorer.
```
Comment (profradha) [inline docs/learn/learn-proxies.md]: You need not use a block explorer. This information can be queried on-chain directly.

```suggestion
can spot the announced extrinsic. Eleanor can check all the proxy call announcements made on her account's behalf on-chain. On Polkadot-JS UI, go to Developer > Storage > Proxy > Announcements to view the list of unexecuted proxy calls on Eleanor's account.
```

File: docs/learn/learn-proxies.md [hunk 5]
```diff
@@ -208,20 +220,38 @@ and the `ProxyDepositFactor` is
...
+the hash of the intended function call in the announcement. Within this time window, the intended
+action may be canceled by accounts that control the proxy. This can be done by the proxy itself
+using the `proxy.removeAnnouncement` extrinsic or by the proxied account using the the
+`proxy.rejectAnnouncement` extrinsic. Now we can use proxies knowing that any malicious actions can
+be noticed and reverted within a delay period. After the time-delay, the proxy can use the
+`proxy.proxyAnnounced` extrinsic to execute the announced call.
+
+:::info
+
+See [this video tutorial](https://youtu.be/LxIW7t15Iuw) to learn how you can setup and use
```
Comment (profradha) [inline docs/learn/learn-proxies.md]: Provided feedback on the video on Element chat.

**Pull request merged**